x-pack/legacy/plugins/security/server/lib/authentication/providers/token.test.ts

```diff
@@ -47,15 +50,6 @@ describe('TokenAuthenticationProvider', () => {
       );
     });
 
-    it('does not handle authentication if state exists, but accessToken property is missing.', async () => {
```

Comment:
__note:__ same here, if we have a state that doesn't have an `accessToken` in it - it's a bug, and I'd rather notice it earlier.